roles/common/tasks/monitoring.yml

```diff
@@ -208,7 +208,8 @@
   file: path=/var/cache/check-log owner=sensu state=directory
 
 - name: raid check
-  sensu_check: name=check-raid plugin=check-raid.sh interval=300 occurrences=1
-               args="-z {{ common.monitoring.sensu_checks.check_raid.criticality }}"
+  sensu_check_dict:
+    name: check-raid
+    check: "{{ sensu_checks.common.check_raid }}"
   notify: restart sensu-client
```

Comment:
Good catch, this was from me mashing the two together. The other handler is what we want in case checks get dropped but sensu-client doesn't actually get installed (for whatever reason).